# ParamyxovirusSlippageEvolution
Supporting information for *Evolutionary history of mRNA editing in the Paramyxovirus phosphoprotein gene*

Contains sequence alignments, .xml files to be run in BEAST2, and the output files of BEAST2 analyses

## Files:


1) **accession_numbers.tsv** | *Accession numbers of all sequences used in the article*

2) **W_functional_alignment.html** | *An html file to visualise the function C-terminal regions of the W protein (Fig 2)*

3) **editsites.tsv** | *Predicted or experimentally determined locations of edit sites in P gene*

4) **edit_sites_alignment.html** | *An html file to visualise edit sites (Fig 6) . Must be run in the same directory as* 
editsites.tsv

5) **PProteins.fasta** | *Predicted or experimentally determined P protein sequences*

6) **PProteins_unq.fasta** | *Predicted or experimentally determined P protein sequences (C-terminal regions only)*

7) **VProteins.fasta** | *Predicted or experimentally determined V protein sequences*

8) **VProteins_unq.fasta** | *Predicted or experimentally determined V protein sequences (C-terminal regions only)*

9) **WProteins.fasta** | *Predicted or experimentally determined W protein sequences*

10) **WProteins_unq.fasta** | *Predicted or experimentally determined W protein sequences (C-terminal regions only)*

11) Phylogenetic tree files as inputs/outputs of BEAST2

- Lprotein_align.fasta

- Lprotein_beast2.log

- Lprotein_beast2.xml

- Lprotein_tree.nexus
	

	
	
	
	
